cores/esp32/HardwareSerial.h

```diff
@@ -165,6 +171,8 @@ typedef enum {
       #define TX1 (gpio_num_t)27
     #elif CONFIG_IDF_TARGET_ESP32S2
       #define TX1 (gpio_num_t)5
+    #elif CONFIG_IDF_TARGET_ESP32C2
+      #define TX1 (gpio_num_t)10
```

Comment:
Pin 9 is already used for I2C. Maybe we can set Serial1 default pins to 10 and 18 (that are free).

```suggestion
    #elif CONFIG_IDF_TARGET_ESP32C2
      #define TX1 (gpio_num_t)18
```